Sources/PasscodeKit/Views/PasscodeInputView.swift

```diff
@@ -86,6 +78,24 @@ public struct PasscodeInputView<Hint>: View where Hint: View {
             }
         }
     }
+    
+    @MainActor
+    private func initialBiometryCheck() async {
```

Comment:
A better name would be `checkBiometry` no? Since this method is not only called initially but also on every `onBiometry` callback.